docs/cli/using-cli.md

```diff
@@ -0,0 +1,81 @@
+# Using the CLI
```

Comment:
I think having everything on the same page might be messy to search by, since this ends getting a bit technical. To avoid cluttering the menu, we could just use a dropdown menu with the options we present on the `How it works`.

Reply:
Absolutely agree with this one.